lib/src/swig/CMakeLists.txt

```diff
@@ -21,8 +21,8 @@ endif()
 
```

Comment:
Why is the main CMakeLists in the swig folder?

Reply:
Copy paste - will remove